jvm-libs/generic/persistence/db/src/testFixtures/kotlin/net/consensys/zkevm/persistence/db/test/CleanDbTestSuiteParallel.kt

```diff
@@ -26,8 +26,6 @@ abstract class CleanDbTestSuiteParallel {
 
   private fun createDataSource(databaseName: String): DataSource {
     return PGSimpleDataSource().also {
-      it.ssl = false
-      it.gssEncMode = "disable"
       it.serverNames = arrayOf(host)
```

Comment:
Reverted the above changes as it didn't fix the tx-exclusion-api integration test random failure issue